src/Microsoft.AspNetCore.Razor.Evolution/Intermediate/CSharpAttributeValueIRNode.cs

```diff
@@ -9,16 +9,14 @@ namespace Microsoft.AspNetCore.Razor.Evolution.Intermediate
 {
     public class CSharpAttributeValueIRNode : RazorIRNode
     {
-        public override IList<RazorIRNode> Children { get; } = EmptyArray;
+        public override IList<RazorIRNode> Children { get; } = new List<RazorIRNode>();
```

Comment:
Also @pranavkm - we were thinking about just splitting the IR back into Tokens + Nodes. Only tokens could represent code with a source mapping, and Nodes could represent context.

But we're not going to block on that right now 👍